src/main.js

```diff
@@ -30,72 +30,150 @@ app.whenReady().then(() => {
+                        { role: "delete" },
+                        { role: "selectAll" },
+                        { type: "separator" },
+                        {
+                            label: "Speech",
+                            submenu: [
+                                { role: "startSpeaking" },
+                                { role: "stopSpeaking" }
+                            ],
+                        },
+                    ]
+                    : [
+                        { role: "delete" },
+                        { type: "separator" },
+                        { role: "selectAll" }
+                    ]),
+            ],
+        },
+        // { role: 'viewMenu' }
+        {
+            label: "View",
             submenu: [
                 {
                     label: "Matches",
                     accelerator: "CmdOrCtrl+M",
                     click: (_, browserWindow) => {
-                        browserWindow.webContents
-                            .send("show-project-pairs-view");
-                    }
+                        browserWindow.webContents.send(
+                            "show-project-pairs-view"
+                        );
+                    },
                 },
                 {
                     label: "Warnings",
                     accelerator: "CmdOrCtrl+W",
                     click: (_, browserWindow) => {
                         browserWindow.webContents.send("show-warnings-view");
-                    }
+                    },
                 },
                 { type: "separator" },
-                { role: "zoomIn", accelerator: "CmdOrCtrl+=" },
-                { role: "zoomOut" },
+                { role: "reload" },
+                { role: "forceReload" },
+                { role: "toggleDevTools" },
+                { type: "separator" },
                 { role: "resetZoom" },
-                { role: "togglefullscreen" },
+                { role: "zoomIn" },
+                { role: "zoomOut" },
                 { type: "separator" },
-                { role: "toggleDevTools" },
-            ]
+                { role: "togglefullscreen" },
+            ],
+        },
+        // { role: 'windowMenu' }
+        {
+            label: "Window",
+            submenu: [
+                { role: "minimize" },
+                { role: "zoom" },
+                ...(isMac
+                    ? [
+                        { type: "separator" },
+                        { role: "front" },
+                        { type: "separator" },
+                        { role: "window" },
+                    ]
+                    : [{ role: "close" }]),
+            ],
         },
         {
-            label: "&Help",
-            click: (_, browserWindow) => {
-                browserWindow.webContents.send("show-help-view");
-            }
-        }
+            role: "help",
+            submenu: [
+                {
+                    label: "Get Help?",
```

Comment:
Is it possible to add a keyboard shortcut here? Before I had ALT+H, which is what it says on the help page.

Reply:
Sure I'll go with alt-h for windows and Cmd+? for mac since these seem to be the standard.

Reply:
Fixed in latest commit.